pkg/wallet/wallet.go

```diff
@@ -259,6 +259,10 @@ func (w *Wallet) InternalizeAction(ctx context.Context, args sdk.InternalizeActi
 		return nil, fmt.Errorf("invalid internalize action args: %w", err)
 	}
 
+	if err := validate.ValidateWalletInternalizeAction(ctx, w.keyDeriver, &wdkArgs); err != nil {
```

Comment:
please call the `validate.ValidInternalizeActionArgs` inside `validate.ValidateWalletInternalizeAction` so here we would have single validation function called.